examples/amp-byside-content.amp.html

```diff
@@ -50,6 +50,7 @@ <h2>Fixed layout content</h2>
 		layout="fixed"
 		width="120"
 		height="40"
+		sandbox="allow-scripts allow-same-origin allow-popups"
```

Comment:
why does this need to be exposed on the component? to the page author, the implementation details that this is an iframe should be irrelevant. Is there  a single default the component can use? If not, under what conditions does the page author need to change them? ( and how would they know what to change them to?)

Reply:
Hi @aghassemi, thanks for your comments. We are going to assume pre-defaults for the component and set iframe attributes automatically.